### ISSUE TITLE
dvrescue capture fails/stuck on timecode jump

### ISSUE THREAD

**Issue by DragonSlayer1514** [mipops/dvrescue#936]
I'm not sure if this is an edge case, but this is the situation which it occurs. I also have rewind on. This is usually due to an accidental overwrite/failed to wipe the tape leaving lingering parts of old clips leading to the middle of a clips where the timecode resets or is shifted. When recording to the end and the shifting timecode causes the recording to fail, that is not an issue. However, when trying to record the start of a clip and with the program either trying to rewind to capture it or starting in the previous clip to try and capture the next clip will cause the recording to fail due to the timecode not being the same. For the first situation I had two other strange behaviors occur as well during the rewinds. First, is it rewinds into the shifted timecode and fails to continue recording. Second, if the clip does not start at timecode 0 (e.g. 00:10:00) and there is blank space before, it continues to rewind trying to search for the previous frame sometimes. Every time the terminal shows dropped frame and stops capturing, but programs does not exit.

**Comment by dericed** [comment]
Hey @DragonSlayer1514, I see a number of issues closed in a quick sequence. Are these issues are resolved?

**Comment by DragonSlayer1514** [comment]
@dericed yes, I ended up using very lousy solution that probably isn't recommended

**Comment by libbyhopfauf** [comment]
@DragonSlayer1514 would you mind installing the latest versions of the daily builds and seeing if that negates the need to use your workaround?
